module/wisp/bin/resources/shaders/deferred_composition.hlsl

```diff
@@ -103,6 +103,9 @@ void main_cs(int3 dispatch_thread_id : SV_DispatchThreadID)
 		//Ao is multiplied with material texture ao, if present
 		ao *= gbuffer_ao;
 
+		//Ao is multiplied with material texture ao, if present
+		ao *= gbuffer_ao;
```

Comment:
Duplicated multiplication? Probably due to wisp already being updated in the latest plugin version?